src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -116,5 +115,5 @@ public void run() {
             }
         });
     }
-
+    
```

Comment:
only whitespace modifications. Revert this file

Reply:
Done